datagen/ScaleTest.md

```diff
@@ -55,7 +55,7 @@ Then a folder with name pattern: `SCALE_<scale factor>_<complexity>_<format>_<da
 will be created under the `<PATH_TO_SAVE_DATA>` that user just provided.
 
 ## Test Query Sets
-To be added.
+Please refer to [Query Sets](../integration_tests/ScaleTest.md#query-sets) in `integration_tests` module for more details.
```

Comment:
Actually this is kind of "broken" for me to see a test suite lies in two different modules. But semantically, the the data gen is about `data gen` and Scale Test is about `scale test`... what do you say? I'm ok to put this into datagen module.